packages/ng-schematics/src/ng-new/index_spec.ts

```diff
@@ -26,6 +26,10 @@ describe("igniteui-angular-schematics", () => {
 			.and.returnValue(Promise.resolve(true)),
 			getProjectLibrary: spyOn(SchematicsPromptSession.prototype, "getProjectLibrary")
 			.and.returnValue((Promise.resolve(mockLibrary))),
+			getProjectTemplate: spyOn(SchematicsPromptSession.prototype, "getProjectTemplate")
+			.and.returnValue(Promise.resolve("empty-page")),
```

Comment:
This might fail, because the name of the empty template was was changed to `'empty'` w/ this PR #623 .

Reply:
It won`t fail as the projectIds are mocked:
`projectIds: ["empty-page"]`